specs/DisableNavigatingBackAndForward.md

```diff
@@ -73,25 +73,36 @@ void WebView_NavigationStarting(object sender, CoreWebView2NavigationStartingEve
 // Enums and structs
 [v1_enum]
 typedef enum COREWEBVIEW2_NAVIGATION_KIND {
+  /// Default kind to cover all unknown kind.
+  /// 
+  /// Note: This field should not be explicitly used by developers, because this kind contains
```

Comment:
If this shouldn't be used by end developers and we can't tell them how to use it, then we shouldn't have it in our API. We also didn't review this with API review board right? We shouldn't add in new changes after the review.

Reply:
This field is taken to avoid compat problems in the future. Do I need to review this with API review board?

Reply:
The API doc is supposed to be in the expected final form before doing API review. We can't modify after review or else we need to do review again.

But in this case, we shouldn't have this enum value. You document it as saying this should not be used by developers.  We should not have an enum value like this.

Reply:
OK. I will update codes according to your suggestion.